src/common/helper/profileUpdate/profile-update.ts

```
@@ -69,6 +69,20 @@ export default class ProfilePopulator {
     return total;
   }
 
+  private convertToTwoDecimalPercentage(input: string | number): string {
+    if (typeof input === 'string') {
+      const parsed = parseFloat(input);
+      if (isNaN(parsed)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)